File: Year 1 Semester 1 (Sem 1)/Computer Lab/Problem Set 2/question1.py
n = int(input())

if n < 0 or n > 100:
    print('Invalid')
else:
    if n > 90:
        print('S')
    elif n > 80:
        print('A')
    elif n > 70:
        print('B')
    elif n > 60:
        print('C')
    elif n >= 50:
        print('D')
    else:
        print('F')

File: Year 1 Semester 2 (Sem 2)/Structured and Object-Oriented Programming/Programming/PS 1/question1.c
// 21BDS0340 Abhinav Dinesh Srivatsa
/*
Declare integers Sum as 0 and N
Read input and assign to N
Calculate Sum as the sum of the second and fourth digit
Display Sum
*/

#include <stdio.h>

int main()
{
    int n, sum = 0;
    scanf("%d", &n);
    sum = (n / 100) % 10 + n % 10;
    printf("%d", sum);
}